[2025-02-13T09:26:13.070+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-13T09:26:13.099+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test_bigquery_connection.check_bigquery manual__2025-02-13T09:26:09.118414+00:00 [queued]>
[2025-02-13T09:26:13.110+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test_bigquery_connection.check_bigquery manual__2025-02-13T09:26:09.118414+00:00 [queued]>
[2025-02-13T09:26:13.111+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-02-13T09:26:13.130+0000] {taskinstance.py:2889} INFO - Executing <Task(BigQueryCheckOperator): check_bigquery> on 2025-02-13 09:26:09.118414+00:00
[2025-02-13T09:26:13.150+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'test_bigquery_connection', 'check_bigquery', 'manual__2025-02-13T09:26:09.118414+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/test-connection-bigquery.py', '--cfg-path', '/tmp/tmpqgla2f45']
[2025-02-13T09:26:13.154+0000] {standard_task_runner.py:105} INFO - Job 7: Subtask check_bigquery
[2025-02-13T09:26:13.157+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1312) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-13T09:26:13.158+0000] {standard_task_runner.py:72} INFO - Started process 1314 to run task
[2025-02-13T09:26:13.258+0000] {task_command.py:467} INFO - Running <TaskInstance: test_bigquery_connection.check_bigquery manual__2025-02-13T09:26:09.118414+00:00 [running]> on host c1819cd5323a
[2025-02-13T09:26:15.448+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='test_bigquery_connection' AIRFLOW_CTX_TASK_ID='check_bigquery' AIRFLOW_CTX_EXECUTION_DATE='2025-02-13T09:26:09.118414+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-13T09:26:09.118414+00:00'
[2025-02-13T09:26:15.452+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-13T09:26:15.487+0000] {baseoperator.py:421} WARNING - BigQueryCheckOperator.execute cannot be called outside TaskInstance!
[2025-02-13T09:26:15.489+0000] {sql.py:796} INFO - Executing SQL check: SELECT 1
[2025-02-13T09:26:15.598+0000] {connection.py:277} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2025-02-13T09:26:15.630+0000] {base.py:84} INFO - Retrieving connection 'google_cloud_default'
[2025-02-13T09:26:15.734+0000] {sql.py:553} INFO - Running statement: SELECT 1, parameters: None
[2025-02-13T09:26:15.737+0000] {bigquery.py:1241} INFO - Inserting job ***_1739438775736473_5089afc024e7a503a1393b79cfe0444f
[2025-02-13T09:26:18.778+0000] {sql.py:799} INFO - Record: [1]
[2025-02-13T09:26:18.780+0000] {sql.py:807} INFO - Success.
[2025-02-13T09:26:18.806+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-13T09:26:18.808+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=test_bigquery_connection, task_id=check_bigquery, run_id=manual__2025-02-13T09:26:09.118414+00:00, execution_date=20250213T092609, start_date=20250213T092613, end_date=20250213T092618
[2025-02-13T09:26:18.897+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-13T09:26:18.936+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
